assets/tests/components/searchResults.test.tsx

```diff
@@ -120,6 +120,72 @@ describe("SearchResults", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("renders vehicles that have logged in within the past 30 minutes", () => {
```

Comment:
A name like "shows the new badge for vehicle that have logged in within the past 30 minutes" would better indicate why we need the test.